tests/test_debug_pipeline_w_sklearn_test.py

```diff
@@ -0,0 +1,1035 @@
+"""
```

Comment:
Are these just the tests copy-pasted from sklearn? That's probably a good idea but it might be nicer to actually import the tests from sklearn and monkeypatch the sklearn pipeline object to your debugpipeline so that changes in the sklearn tests automatically will be captured here as well. 

If you also think this is a good idea I'd be open to help you / add it myself

Reply:
They are from sklearn yes, but I had to change one test. (Actually two because `make_debug_pipeline` is not implemented yet, but that is for future worry.) One test fails `test_set_pipeline_step_none` because there is a new property, namely the `log_callback`.

Reply:
Ah yes that makes sense